src/tests/degreeQuery.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-console */
+import { expect } from 'chai';
+import {} from 'mocha';
+import supertest from 'supertest';
+
+describe('GraphQL', () => {
+    let request: any;
+
+    beforeEach(() => {
+        const url = 'http://localhost:3001';
+
+        request = supertest(url);
+    });
+
+    it('Returns degree with id = 1', done => {
+        request
+            .post('/graphql')
+            .send({
+                query: '{ degree(id: 1) { id nameJa nameEn abbreviation }}'
+            })
+            .expect(200)
+            .end((err: any, res: any) => {
```

Comment:
nit: we could use `await` here instead of the `.end()` callback

Reply:
We could, but matching the current pattern of the tests. Let's do a later refactor